src/components/table/Table.tsx

```diff
@@ -34,6 +34,7 @@ export interface TableProps extends React.HTMLAttributes<HTMLTableElement> {
   compact?: boolean;
   interactive?: boolean;
   striped?: boolean;
+  stickyHeader?: boolean;
```

Comment:
This should only be in the props of the `TableHeader` component and be called `sticky`.